lib/Image/Chromium.php

```diff
@@ -31,15 +31,18 @@ class Chromium
 {
     public static function isSupported(): bool
     {
+        $return = false;
         if (class_exists(BrowserFactory::class)) {
-            try {
-                return (new Connection(\Pimcore\Config::getSystemConfiguration('chromium')['base_url']))->connect();
-            } catch (BrowserConnectionFailed $e) {
-                //websocket fails, then we check binary
-                return (bool) self::getChromiumBinary();
+            $return = (bool) self::getChromiumBinary();
+            if (!$return){
+                try {
+                    $return = (new Connection(\Pimcore\Config::getSystemConfiguration('chromium')['base_url']))->connect();
+                } catch (BrowserConnectionFailed $e) {
```

Comment:
can't we just make a finally here instead of the catch?

Reply:
thank you, that would also be fine. 
But at the moment i am not even sure of this `try` block used to be `falsy` on error/exceptions etc..and on how it would perform 🤔 any suggestions?